src/processor/operator/ddl/create_node_table.cpp

```diff
@@ -8,7 +8,7 @@ namespace kuzu {
 namespace processor {
 
 void CreateNodeTable::executeDDLInternal() {
-    auto newTableID = catalog->addNodeTableSchema(tableName, primaryKeyIdx, properties);
+    auto newTableID = catalog->addNodeTableSchema(tableName, primaryKeyIdx, std::move(properties));
```

Comment:
Although DDL will be executed only once, it's safer to copy here than more I think.